I'm sorry, but an uncaught exception occurred.

While running game code:
  File "renpy/common/00start.rpy", line 195, in script
    python:
Exception: Possible infinite loop.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "renpy/common/00start.rpy", line 195, in script
    python:
  File "g:\renpy\renpy-8.1.1-sdk\renpy\execution.py", line 61, in check_infinite_loop
    raise Exception("Possible infinite loop.")
Exception: Possible infinite loop.

Windows-10-10.0.22621 AMD64
Ren'Py 8.1.1.23060707
专舟历险记 0.11
Tue Sep 23 13:38:16 2025
